### ISSUE TITLE
Audit #1 – General Overview when logged in – Route change don't cause live region updates

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4346]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
Unsure about this one (WCAG 4.1.3, BITV 4.1.3a). In the German language support document of this test step there are concrete examples of state changes that are caused in an asynchronous manner.

In my perspective all status messages fired are valuable (one could discuss about setting everything "assertive") but , quote: 'the message "provides information to the user on the success or results of an action, on the waiting state of an application, on the progress of a process, or on the existence of errors;"' (https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html, Intent) is not fully taken care of because of missing focus management of live region activation on route change.

# Remediation
* Possibly a new live region with `aria-live="polite"` or `role="status"` is suitable for most of the statusses and route change, because `aria-live="assertive"` is too aggressive and should actually only be used for fatal errors and the like.

# Update: Rule of thumb
**A message via live region (or focus management) should occur when the asynchronous change in a document happens ABOVE the text node that triggered it (speaking of position in the DOM).**
